kore/src/Kore/Repl.hs

```diff
@@ -29,6 +29,8 @@ import           Control.Monad.State.Strict
 import           Control.Monad.State.Strict
                  ( lift )
 import qualified Data.Graph.Inductive.Graph as Graph
+import           Data.Map.Strict
+                 ( empty )
```

Comment:
We usually import the `Map` type unqualified, and the whole module qualified, like so:
```
import Data.Map.Strict (Map)
import qualified Data.Map.Strict as Map
```

And then we use `Map.empty`, etc. It makes it easier to know what `empty` means.